lib/racecar/consumer.rb

```diff
@@ -27,7 +27,15 @@ def subscriptions
       # @param additional_config [Hash] Configuration properties for consumer.
       #   See https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md
       # @return [nil]
-      def subscribes_to(*topics, start_from_beginning: true, max_bytes_per_partition: 1048576, additional_config: {})
+      def subscribes_to(
+        *topics,
+        start_from_beginning: true,
+        max_bytes_per_partition: 1048576,
+        additional_config: {},
+        parallel_workers: nil
```

Comment:
I don't think it makes sense to make this configuration part of the `subscribes_to` method – consumer classes can subscribe to multiple topics, but the parallelism is tied to the consumer, not the topic. The class level accessor should suffice; `class MyConsumer < ...; self.parallel_workers = 5; end`.